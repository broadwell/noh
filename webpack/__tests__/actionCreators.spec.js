import {
  setCurrentTime,
  setIsPlaying,
  setStartTime,
  setCurrentPhraseID,
  setScoreToggles,
  setNarrativeTab
} from "../actionCreators";

test("setIsPlaying creates the correct payload", () => {
  expect(setIsPlaying(true)).toMatchSnapshot();
});

test("setCurrentTime creates the correct payload", () => {
  expect(setCurrentTime(30)).toMatchSnapshot();
});

test("setStartTime creates the correct payload", () => {
  expect(setStartTime(30)).toMatchSnapshot();
});

test("setCurrentPhraseID creates the correct payload", () => {
  expect(setCurrentPhraseID("II/1")).toMatchSnapshot();
});

test("setScoreToggles creates the correct payload", () => {
  expect(
    setScoreToggles({ isTextOn: false, isBeatOn: false })
  ).toMatchSnapshot();
});

test("setNarrativeTab creates the correct payload", () => {
  expect(setNarrativeTab(1)).toMatchSnapshot();
});
