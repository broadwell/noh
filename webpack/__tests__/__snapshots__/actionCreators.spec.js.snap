// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`setCurrentTime creates the correct payload 1`] = `
Object {
  "payload": 30,
  "type": "SET_CURRENT_TIME",
}
`;
